newrelic/structures_newrelic_synthetics_multilocation_alert_condition.go

```diff
@@ -28,7 +28,12 @@ func expandMultiLocationSyntheticsCondition(d *schema.ResourceData) (*alerts.Mul
 
 	var entities []string
 	for _, x := range d.Get("entities").([]interface{}) {
-		entities = append(entities, getMonitorID(x.(string)))
+		monitorID, err := getMonitorID(x.(string))
+
+		if err != nil {
+			return nil, err
+		}
+		entities = append(entities, monitorID)
```

Comment:
I just discovered from a test that we might need to further correct this logic to handle an error if "x" is not convertible into a string. If this is not implemented, upon adding a list of empty strings to `entities` in the Terraform Configuration, the following is seen 

```
Stack trace from the terraform-provider-newrelic plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 87 [running]:
github.com/newrelic/terraform-provider-newrelic/v2/newrelic.expandMultiLocationSyntheticsCondition(0x10?)
        /Users/x/myWork/src/github.com/terraform-provider-newrelic/newrelic/structures_newrelic_synthetics_multilocation_alert_condition.go:31 +0x368
```

Let's work on this tomorrow.